lib/knackhq/client.rb

```diff
@@ -16,62 +16,64 @@ def objects
       hash_request = request
                      .objects
                      .get
-                     .to_h[:objects]
-      return [] if hash_request.empty?
-      symbolize_hash_keys!(hash_request)
+      payload = payload_hash(hash_request)
+      translate_payload(payload) { payload[:objects] }
     end
 
     def object(key)
-      hash_request = request.objects.get(key).to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:object]['fields']
-      symbolize_hash_keys!(transform_hash_keys)
+      hash_request = request.objects.get(key)
+      payload = payload_hash(hash_request)
+      translate_payload(payload) { payload[:object][:fields] }
     end
 
     def fields(object)
       hash_request = request_object(object, :fields)
```

Comment:
May I suggest replacing this with a direct call to request since the request_object only appears to be used once.